README.md

```diff
@@ -1,20 +1,26 @@
-# KAM (Kubernetes Application Manager) CLI
+# GitOps Application Manager
 
-KAM CLI provides command line interface to bootstrap GitOps and perform other operations.  See why and how [GitOps for Application Delivery](./docs/README.md) can help your business to streamline your application delivery.
+The GitOps Application Manager provides command line interface to bootstrap GitOps and perform other operations.  See why and how [GitOps for Application Delivery](./docs/README.md) can help your business streamline your application delivery.
 
 ## CLI Reference
 
 [Comand Line Reference](./docs/commands/README.md)
 
-## Setup for DevConsole
+## Getting Started
 
-[An example setup steps which allows OpenShift DevConsole to display GitOps](./docs/devconsole)
```

Comment:
This is a link to show an example to get your gitops to be shown in UI.   Should we keep it?